mllib-local/pom.xml

```diff
@@ -62,6 +62,15 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.json4s</groupId>
+      <artifactId>json4s-jackson_${scala.binary.version}</artifactId>
+      <version>3.2.10</version>
```

Comment:
I have another JIRA to do so. We also want to consolidate the version of breeze. https://issues.apache.org/jira/browse/SPARK-14612

If you want, I can do it in this PR.